### PR TITLE
new:dev: Piazza Consolidate-different entry types and fields in Bib (work for #110)

### PR DESCRIPTION
**Pull request by vibhatha**
Adding a new post regarding bib entries

### CODE REVIEW

File: docs/source/faq.rst [hunk 1]
```diff
@@ -581,26 +596,28 @@ should show the changes only in refs.bib
...
-The changes in the file to resolve merge conflict automatically goes to the original pull request and the pull request can be merged automatically
+The changes in the file to resolve merge conflict automatically goes
+to the original pull request and the pull request can be merged
+automatically
 
 
 
-Cheat sheet for Linux commands
-------------------------------
+Cheat sheet for Linux commands ------------------------------
```
Comment (badi) [inline docs/source/faq.rst]: This will break the rst sectioning

**Comment by vibhatha** [comment]
Yes I didn't change anything. Just pasted the commands which were in your
posts. I didn't change anything.

With Regards,
Vibhatha Abeykoon

**Comment by laszewsk** [comment]
than you have not used propper rst with ::

**Comment by vibhatha** [comment]
You mean when embedding computer codes? Is there a specific way to do this
?

With Regards,
Vibhatha Abeykoon

**Comment by badi** [comment]
@vibhatha in RST, the following:

---

```rst
Example running in a shell prompt:

::

  $ echo "hello world"
```

---

would render as:

---

Example running in a shell prompt:

```
$ echo "hello world"
```

---

**Pull request closed without merging**